analytics/src/main/java/com/segment/analytics/Analytics.java

```diff
@@ -438,6 +440,9 @@ void recordScreenViews(Activity activity) {
 
   @Private
   void runOnMainThread(final IntegrationOperation operation) {
+    if (shutdown) {
```

Comment:
It's not necessary, and one of the tests uses this to verify the activity callbacks don't propagate. I was just being extra defensive as runOnMainThread _could_ be called from a few other places.